### PR TITLE
Config file

### PR DESCRIPTION
**Pull request by mahipal-z**
With this update: 
A. Client can upload CSV files in any folder with no naming restriction, in any nested folder structure within source bucket.
B. Upon updating the processed file location in the notebook_configuration.config file, client can run jupyter notebook to visualize insights in the sagemaker instance.
C. Can see the model explanations in the web application.

### CODE REVIEW

File: Analytics/rough_sheet.py
Comment (kzaamout) [inline Analytics/rough_sheet.py:1]: What purpose does this file serve? I find no documentation in it. 
Please go over all your code files and add documentation at the top of the files explaining what they do.

File: CloudFormation/notebook_configuration.config
Comment (kzaamout) [inline CloudFormation/notebook_configuration.config:1]: What purpose is this file? not seeing it being used anywhere?

**Comment by kzaamout** [comment]
Don't commit any of the .ipynb_checkpoint folders. they are generated.

under Datasets, you have a blank file and a .sav file. remove them. the first is because it's empty. the second because your code generates it. If you have a case against removing them, please share it with me.

**Comment by mahipal-z** [comment]
I have made the requested changes with another branch. This branch is redundant now.

**Pull request closed without merging**